docs/settings/telemetry-settings.asciidoc

```diff
@@ -4,32 +4,30 @@
 <titleabbrev>Telemetry settings</titleabbrev>
 ++++
 
-By default, Usage Collection (also known as Telemetry) is enabled. This
-helps us learn about the {kib} features that our users are most interested in, so we
```

Comment:
mostly removing Kibana features because we also collect telemetry about other Elastic products.

Reply:
Fair point!

@cdanderson98, what do you think?

Reply:
@gchaps, I can see "Usage Data" is only used one (when referring to Kibana's Advanced Settings section.

If that's fine, I'll merge this docs PR so we can backport it to all previous versions, and we can update that to "Usage Collection" for the next release.

Re Usage Collection vs. Telemetry. I tend to prefer Usage Collection because Telemetry is too broad (it can also refer to Observabilty metrics). However, removing `telemetry` from the settings will introduce Breaking changes. That's the main reason we still handle 2 terms to refer to the same concept.

I hope it makes sense.